app/components/password_confirmation_component.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= content_tag(:'lg-password-confirmation', **tag_options) do %>
+  <%= render ValidatedFieldComponent.new(
+        form: form,
+        name: :password,
+        type: :password,
+        label: default_label,
+        required: true,
+        **field_options,
+        input_html: field_options[:input_html].to_h.merge(
+          id: input_id,
+          class: ['password-confirmation__input', *field_options.dig(:input_html, :class)],
+        ),
+      ) %>
+  <%= render ValidatedFieldComponent.new(
+        form: form,
+        name: :password_confirmation,
+        type: :password_confirmation,
+        label: confirmation_label,
+        required: true,
+        **field_options,
+        input_html: field_options[:input_html].to_h.merge(
+          id: input_confirmation_id,
+          class: ['password-confirmation__input-confirmation', *field_options.dig(:input_html, :class)],
+        ),
+        error_messages: {
+          valueMissing: t('components.password_confirmation.errors.empty'),
+        },
+      ) %>      
+  <input
+    id="<%= toggle_id %>"
+    type="checkbox"
+    class="usa-checkbox__input password-confirmation__toggle"
+    aria-controls="<%= input_id %>"
```

Comment:
This checkbox controls both inputs, and `aria-controls` can include multiple values as a space-separated list, so I wonder if we should include all of the fields which would be affected by it.

```suggestion
    aria-controls="<%= input_id %> <%= input_confirmation_id %>"
```